src/main/java/edu/kis/vh/nursery/list/Node.java

```diff
@@ -2,8 +2,8 @@
 
 public class Node {
 
-	public final int value;
-	public Node prev, next;
+	protected final int value;
+	protected Node prev, next;
```

Comment:
encapsulation should be done by reducing visibility to private and introducing getters and setters